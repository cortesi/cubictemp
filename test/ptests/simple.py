mdict = {
	"foo": "one",
	"bar": "two",
	"boo": "three",
	"mdict": {"entry": "foo"}
}
