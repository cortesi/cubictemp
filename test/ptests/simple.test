@!foo!@
@!foo!@|@!bar!@|@!boo!@
@!foo!@
@!bar!@
@!boo!@
@!mdict["entry"]!@
