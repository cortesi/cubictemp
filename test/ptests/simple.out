one
one|two|three
one
two
three
foo
